src/pages/ForgotScreen.jsx

```diff
@@ -0,0 +1,61 @@
+// Import necessary dependencies
+import React from "react";
+import ForgotPasswordImage from "../assets/images/Forgot-password.png";
+import "/home/dckap/Palli-Application/Palli-Application/src/assets/css/forgotPage.css";
+
+// Define a functional component for the Forgot Password page
+const Forgotpassword = () => {
+  return (
+    <div className="forgot-container">
+      {/* Left side container with an image */}
+      <div className="left-side">
+        <img src={ForgotPasswordImage} alt="ForgotPasswordImage" />
```

Comment:
ForgotPasswordImage -> mention the relative path here -> src/assets/images/xxx.png